common/devdoc/interlocked_hl_requirements.md

```diff
@@ -63,22 +63,40 @@ MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForVal
 ```
 
 `InterlockedHL_WaitForValue` waits for the value at a given address to be equal to a target value.
+[Note: `address_to_check` refers to `address`, `value_to_wait` refers to `value`, and `timeout_ms` refers to `milliseconds`]
```

Comment:
There is no reason to "alias" here, if we're changing the argument names, then please let's do so by renaming them in line 62 (and in the API description and the header file).
Having an indirection layer like this only makes more work when reading it.

Reply:
Not fixed:

<img width="1520" alt="image" src="https://github.com/user-attachments/assets/670db983-3b2f-41bf-8c5f-0dc3f9ec6a9a">